### ISSUE TITLE
Small anterior pituitary lobe

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9901]
**Preferred term label:**
also
- Ectopic posterior pituitary lobe
- Absent pituitary stalk
- Pituitary stalk interruption 

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
620303 
https://pubmed.ncbi.nlm.nih.gov/28402530/

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
see Anterior pituitary hypoplasia HP:0010627

**Comment by pnrobinson** [comment]
added
